python/tvm/meta_schedule/task_scheduler/gradient_based.py

```diff
@@ -15,9 +15,11 @@
 # specific language governing permissions and limitations
 # under the License.
 """Gradient Based Task Scheduler"""
+import logging
 from typing import TYPE_CHECKING, List, Optional
 
 from tvm._ffi import register_object
+from tvm.meta_schedule.utils import make_logging_func
```

Comment:
use relative import:

```suggestion
from ..utils import import make_logging_func
```